com.gooseberrydevelopment.pinepods.yml

```diff
@@ -0,0 +1,38 @@
+id: com.gooseberrydevelopment.pinepods
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: app
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: pinepods-client
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/madeofpendletonwool/PinePods/releases/download/0.7.1/Pinepods_0.7.1_amd64.deb
+        sha256: e17dfad78f8a3e88b7beab9a103d34270e25de527db375a4a2d5e248888c8c6a
+        only-arches: [x86_64]
+        dest-filename: pinepods.deb
+      - type: file
+        url: https://github.com/madeofpendletonwool/PinePods/releases/download/0.7.1/Pinepods_0.7.1_arm64.deb
+        sha256: a14e15df40b8d2646ed118f041ee4d2220cc5b324d0dc533a405ac762bbc5ecb
+        only-arches: [aarch64]
+        dest-filename: pinepods.deb
```

Comment:
you removed the metainfo but it needs to in the deb or as a file source with URL and sha256 set.

then needs to be installed in `build-commands`.

Reply:
It seems to be correct now

```shell
➜  flathub git:(pinepods-submission-0.7.1) ✗ flatpak run --command=ls com.gooseberrydevelopment.pinepods /app/share/metainfo
com.gooseberrydevelopment.pinepods.metainfo.xml
➜  flathub git:(pinepods-submission-0.7.1) ✗ flatpak run --command=cat com.gooseberrydevelopment.pinepods /app/share/metainfo/com.gooseberrydevelopment.pinepods.metainfo.xml
<?xml version="1.0" encoding="UTF-8"?>
<component type="desktop-application">
  <id>com.gooseberrydevelopment.pinepods</id>
  <metadata_license>CC0-1.0</metadata_license>
  <project_license>GPL-3.0-only</project_license>
``` 

I do for some reason get this though. I'm not sure if that's because I'm running the linting locally perhaps?

```shell
➜  flathub git:(pinepods-submission-0.7.1) ✗ flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions repo repo
{
    "errors": [
        "appstream-external-screenshot-url",
        "appstream-remote-icon-not-mirrored"
    ],
    "info": [
        "appstream-remote-icon-not-mirrored: Remote icons are not mirrored to Flathub. Please see the docs for more information",
        "appstream-external-screenshot-url: Screenshots are not mirrored to https://dl.flathub.org/media"
    ],
    "message": "Please consult the documentation at https://docs.flathub.org/docs/for-app-authors/linter"
}
``` 
As I see the docs show that error can sometimes be resolved by rebuilding again.